python/repositories.bzl

```diff
@@ -472,6 +486,19 @@ def python_register_toolchains(
 
     toolchain_repo_name = "{name}_toolchains".format(name = name)
 
+    bazel_major = int(native.bazel_version.split(".")[0])
```

Comment:
I think this is broken for bazel built from head.

FYI https://github.com/aspect-build/bazel-lib/blob/4edb5018e89cd56515fed269056429c353cd7c43/lib/private/utils.bzl#L146

Reply:
Yes, that should be fixed by #1063 (let me know otherwise)

> aspect utils.bzl

Oh, nice find. That would probably also work with a custom Bazel 5.x build, too.